packages/web-app-files/src/views/SharedWithMe.vue

```diff
@@ -256,3 +457,28 @@ export default {
   }
 }
 </script>
+
+<style>
+.centered {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+}
+.shares-bar {
+  display: flex;
+  flex-direction: row;
+  align-items: baseline;
+}
+
+#pending-highlight {
+  background-color: aliceblue;
```

Comment:
Could you check if there is a css variable from our color design tokens? Ideally we don't to have explicit colors anymore in the web code, so that theming works properly. https://owncloud.design/#/Design%20Tokens